Dockerfile

```diff
@@ -5,11 +5,13 @@ FROM ${BASE_DOCKER} AS gradleBuild
 LABEL stage=gatkIntermediateBuildImage
 ARG RELEASE=false
 
-RUN ls .
+#unnecessary
+#RUN ls .
```

Comment:
Can you restore this `ls` command at the beginning of the first `RUN` statement? It is theoretically useful for debugging logs of the docker build when things go wrong.

Reply:
Sounds good.